src/ontology/fypo-odk.yaml

```diff
@@ -15,17 +15,33 @@ release_artefacts:
   - full
 primary_release: full
 import_group:
+  use_base_merging: TRUE
+  exclude_iri_patterns:
+    - <http://purl.obolibrary.org/obo/UBERON_*>
```

Comment:
So here, the idea is to list the patterns that we do _not_ want in the merged import module. The way to proceed here is to actually run the imports pipeline once, and check the ontology in Protégé for “dangling root classes” that should not be there (classes that you never use anywhere in the ontology and that are only there because they were brought in by some imports).

From what I see with the tests I ran, in addition to `UBERON_*` you can also add:

```
    - <http://purl.obolibrary.org/obo/CARO_*>
    - <http://purl.obolibrary.org/obo/NCBITaxon_*>
    - <http://purl.obolibrary.org/obo/OBI_*>
```